### ISSUE TITLE
Fix lint errors raised in flake8 3.5

### ISSUE THREAD

**Issue by jhnlsn** [capitalone/cloud-custodian#1785]

**Comment by jhnlsn** [comment]
```c7n/cli.py:126:5: E722 do not use bare except'
c7n/cli.py:148:5: E722 do not use bare except'
c7n/commands.py:453:5: E722 do not use bare except'
c7n/handler.py:54:13: E722 do not use bare except'
c7n/mu.py:936:9: E722 do not use bare except'
c7n/mu.py:942:9: E722 do not use bare except'
c7n/output.py:105:9: E741 ambiguous variable name 'l'
c7n/output.py:107:13: E741 ambiguous variable name 'l'
c7n/tags.py:286:9: E722 do not use bare except'
c7n/utils.py:404:9: E722 do not use bare except'
c7n/filters/offhours.py:341:9: E722 do not use bare except'
c7n/resources/awslambda.py:245:13: E722 do not use bare except'
c7n/resources/ecr.py:108:13: E722 do not use bare except'
c7n/resources/glacier.py:127:13: E722 do not use bare except'
c7n/resources/kms.py:254:13: E722 do not use bare except'
c7n/resources/sns.py:97:13: E722 do not use bare except'
c7n/resources/sqs.py:129:13: E722 do not use bare except'```

**Comment by kapilt** [comment]
working on it now

**Comment by kapilt** [comment]
merged in #1786
